src/test/regress/expected/multi_prepare_plsql.out

```diff
@@ -509,119 +478,570 @@ SELECT * FROM temp_table ORDER BY key, value;
    4 |      
    5 |    50
    5 |      
-(16 rows)
+(22 rows)
 
 -- check router executor select
-CREATE OR REPLACE FUNCTION partition_column_select(key_arg int) RETURNS TABLE(key int, value int) AS $$
+CREATE FUNCTION router_partition_column_select(key_arg int)
+	RETURNS TABLE(key int, value int) AS $$
 DECLARE
 BEGIN
     RETURN QUERY
 	SELECT
-		temp_table.key,
-		temp_table.value
+		plpgsql_table.key,
+		plpgsql_table.value
 	FROM
-		temp_table
+		plpgsql_table
 	WHERE
-		temp_table.key = key_arg
+		plpgsql_table.key = key_arg
 	ORDER BY
 		key,
 		value;
 END;
 $$ LANGUAGE plpgsql;
-SELECT partition_column_select(1);
- partition_column_select 
--------------------------
+SELECT router_partition_column_select(1);
+ router_partition_column_select 
+--------------------------------
  (1,10)
  (1,)
 (2 rows)
 
-SELECT partition_column_select(2);
- partition_column_select 
--------------------------
+SELECT router_partition_column_select(2);
+ router_partition_column_select 
+--------------------------------
  (2,20)
  (2,)
 (2 rows)
 
-SELECT partition_column_select(3);
- partition_column_select 
--------------------------
+SELECT router_partition_column_select(3);
+ router_partition_column_select 
+--------------------------------
  (3,30)
  (3,)
 (2 rows)
 
-SELECT partition_column_select(4);
- partition_column_select 
--------------------------
+SELECT router_partition_column_select(4);
+ router_partition_column_select 
+--------------------------------
  (4,40)
  (4,)
 (2 rows)
 
-SELECT partition_column_select(5);
- partition_column_select 
--------------------------
+SELECT router_partition_column_select(5);
+ router_partition_column_select 
+--------------------------------
  (5,50)
  (5,)
 (2 rows)
 
--- 6th execution is failing. We don't want to run the failing test because of
--- changing output. After implementing this feature, uncomment this.
--- SELECT partition_column_select(6);
+-- FIXME: 6th execution is failing. We don't want to run the failing test
```

Comment:
Trying to enable this test showed me 16 messages like

```
WARNING:  there is no parameter $1
CONTEXT:  while executing command on localhost:57638
```

Is there a way we can error out earlier that would help the user understand what's going on and let us enable these tests?